src/utils/Logger.ts

```diff
@@ -182,7 +262,17 @@ class Logger {
       warn: console.info,
     }
 
-    const writeLog = (level: string, message: string) => {
+    const writeLog = async (level: string, message: string) => {
+      // If log folder not present create it
+      if (!(await RNFS.exists(this.getReactNativeLogsDir()))) {
+        await RNFS.mkdir(this.getReactNativeLogsDir())
```

Comment:
this could throw, it's a toctou bug.  you should handle the error, or have this code as part of log initialization (where's it's guaranteed to be serialized)

Reply:
The problem I am seeing testing the change with moving it to the log initialization, out of `writeLog` up into the `overrideConsoleLogs` call, is that we miss some of the early logs from the the app start.

Reply:
The first line in the daily logs should be `Reactotron Configured` from `index.js` if it's not that line we're missing logs from app start. This can be tested by creating a new app install locally and navigating to developer settings  and selecting debug.